src/descriptor/mod.rs

```diff
@@ -29,12 +29,15 @@ use std::{
     str::{self, FromStr},
 };
 
-use bitcoin::blockdata::{opcodes, script};
 use bitcoin::hashes::hash160;
 use bitcoin::hashes::hex::FromHex;
 use bitcoin::secp256k1;
 use bitcoin::util::bip32;
 use bitcoin::{self, Script};
+use bitcoin::{
+    blockdata::{opcodes, script},
+    util::psbt::serialize::Serialize,
```

Comment:
You definitely do not want this serialize function!

Reply:
EEK! I blame Rust Analyzer!!!

Pushed a commit to serialize public keys using `pubkey.key.serialize()` and removed that import.

Reply:
Perfect!